tools/openvino_dev/CMakeLists.txt

```diff
@@ -54,6 +54,35 @@ foreach(EXTRAS IN LISTS EXTRAS_LIST)
             ${OV_CPACK_COMP_OPENVINO_DEV_REQ_FILES_EXCLUDE_ALL})
 endforeach()
 
+# check __init__.py files alignment
+
+function(ov_check_init_files_alignment)
+    # check the files in pairs
+    list(APPEND init_files
+        "${OpenVINO_SOURCE_DIR}/tools/mo/openvino/__init__.py"
+        "${OpenVINO_SOURCE_DIR}/tools/pot/openvino/__init__.py"
+        "${OpenVINO_SOURCE_DIR}/tools/ovc/openvino/__init__.py"
+        "${OpenVINO_SOURCE_DIR}/tools/benchmark_tool/openvino/__init__.py"
+        "${OpenVINO_SOURCE_DIR}/tools/openvino_dev/src/openvino/__init__.py")
```

Comment:
should we also include files from OMZ?

Reply:
That's a tough call I guess. I believe that if anything, it's OMZ which should be checking their alignment with OpenVINO. If we include the files here, then in case there is a misalignment:
1. We'll observe it only when bumping OMZ submodule
2. We'll have to create a ticket to OMZ or make the changes ourselves
3. We'll have to wait with the OMZ bump for the OMZ to be fixed (otherwise our CMake fails)

Whereas if the alignment is being checked in OMZ, a PR misaligning the init files could never be merged in the first place. Additionally, I believe we are going to eventually move away from testing OMZ/NNCF in OV CI (cc @mlukasze).

@jiwaszki what do you think?

Reply:
I agree, just wanted to understand the position.

Reply:
Point (3) is the most important one. They should check it on their side to be compatible with upstream (openvino).